### DISCUSSION TITLE
JVCL  won't load in Delphi 12

### DISCUSSION THREAD

**Discussion by pka4916** [project-jedi/jvcl#172] [Q&A]
JVCL latest does install  in delphi 12,  but it won't load.   it;s telling me that it cannot load the bpl's from JVCL.

**Comment by obones** [comment]
What error message?

**Comment by pka4916** [comment]
![Screenshot 2023-11-09 180003](https://github.com/project-jedi/jvcl/assets/34923634/780442be-6b5b-4786-8544-3ad3baaf78a6)

here is a screen.

**Comment by pka4916** [comment]
![Screenshot 2023-11-09 180312](https://github.com/project-jedi/jvcl/assets/34923634/45b44f3d-b145-4692-b623-c0227465ec5f)

nothing of jvcl is loaded..

**Comment by pka4916** [comment]
looks like certain files are not compiled at all

the yellow ones,   Once I compile them,  I can check the packages.   


![Screenshot 2023-11-09 182856](https://github.com/project-jedi/jvcl/assets/34923634/31843454-6fd8-42f6-ac81-8830496658ba)

**Comment by pka4916** [comment]
noticed that  you cannot check  "Build Packages" either...    that will result in missing RES files.

**Comment by pka4916** [comment]
could this be it?

![Screenshot 2023-11-09 183549](https://github.com/project-jedi/jvcl/assets/34923634/2262e352-81f4-4de1-bee6-1679b748020a)

**Comment by pka4916** [comment]
it's working now :)

guess it got missed in the inc  file

**Comment by obones** [comment]
It should just be the PATH environment variable settings.
Note that the Delphi installer does the modification but it won't apply to the system until after having rebooted.

**Comment by jvmbatista** [comment]
Hello, just going to hijack this so I don't create a new thread, any ideia when it will be available in GetIt?

Thanks

**Reply by obones** [reply]
We do not have any control whatsoever on GetIT content, it's all managed by Embarcadero and they usually don't even contact us when they make it available.

**Reply by jvmbatista** [reply]
Wasn't expecting that one... Thanks ;)

I'll try to get it installed manually. I'll be back if I have any issue :P

**Comment by checkdigits** [comment]
Ironically @obones pointed me to this thread after I reached out to him about putting JVCL and JCL into RAD Studio 12's GetIt list! 🤣 

We will get this updated **ASAP** so that RAD Studio 12 lists the JVCL and JCL appropriately.

Any problems please tag me on this thread or email me directly at ian.barker@embarcadero.com

**Reply by jvmbatista** [reply]
We're finally trying to move to 12 and luckily I've had a bunch of other stuff to do so I've been kinda waiting for it

**Reply by jvmbatista** [reply]
@checkdigits 
Available and installed. Looks like JVCL installation stays behind RAD window. Also, if you start with JVCL it looks like it installs both JVCL and JCL.

**Reply by checkdigits** [reply]
Did you manage to get it installed ok despite that?

I can report the issues to the GetIt team if they are reproducible.

**Reply by obones** [reply]
Despite what?
The JVCL is built upon the JCL so it's quite expected to have both installed when you only install JVCL.
As to why the installer stays in the back, I have no idea but I would suggest using Alt-Tab to bring it back to front

**Reply by jvmbatista** [reply]
@checkdigits, everything looks like it was installed correctly. Still have a couple of addons to go through and then I'll begin what I'm almost certain that will be a painful process and update my projects.

@obones, last time I installed it was on XE3 and if I recall correctly, I had to install both. Probably I was doing it wrong back then xD. 
It makes sense that if you're installing jvcl that it also instals jcl.

**Reply by checkdigits** [reply]
> Despite what?

Despite the JVCL installation saying behind the RAD Studio window...
